metadata-etl/src/main/resources/jython/HiveTransform.py

```diff
@@ -12,10 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #
 
-import json
-import datetime
-import sys, os
-import time
+import json, sys, os, time
```

Comment:
Generally I'd prefer to keep each import on separate lines and alphabetically ordered, which makes diff easier when they're added & removed.